modules/minio/go.mod

```diff
@@ -0,0 +1,51 @@
+module github.com/testcontainers/testcontainers-go/modules/minio
+
+go 1.20
+
+require github.com/testcontainers/testcontainers-go v0.27.0
```

Comment:
I believe you need to run `go mod tidy` from here to include the minio client as a dependency

Reply:
Thanks this has just made me realise I accidentally imported some code from a personal library that I use for testing. So will need to clean and replace that part of the test